src/features/featureQuery.js

```diff
@@ -435,6 +532,26 @@ FeatureQuery.prototype.lookupFeature = function (query) {
                         }));
                     }
                     break;
+                case '51':
+                    substitution = lookup(contextParams);
+                    if (Array.isArray(substitution) && substitution.length) {
```

Comment:
Are there cases where substitution is not an array or might be empty? Or should that theoretically never happen in a well-formed font and this is just to catch any malformed ones?